addons/mail/static/src/components/activity/activity.xml

```diff
@@ -64,8 +67,8 @@
                                 <div t-if="activityView.activity.creator" class="d-md-table-row mb-3">
                                     <div class="d-md-table-cell font-weight-bold text-md-right m-0 py-md-1 px-md-4">Created</div>
                                     <div class="o_Activity_detailsCreation d-md-table-cell py-md-1 pr-4">
-                                        <t t-esc="activityView.formattedCreateDatetime"/>, by
-                                        <img class="o_Activity_detailsUserAvatar o_Activity_detailsCreatorAvatar" t-attf-src="/web/image/res.users/{{ activityView.activity.creator.id }}/avatar_128" t-att-title="activityView.activity.creator.nameOrDisplayName" t-att-alt="activityView.activity.creator.nameOrDisplayName"/>
+                                        <t t-esc="activityView.formattedCreateDatetime"/>, <br class="d-md-none"/>by
```

Comment:
Don't use `d-none`, prefer not rendering useless elements: `<br class="d-md-none"/>` -> `<br t-if="messaging.device.isMobile"/>`